### PR TITLE
Gets the number of rows

### PR DESCRIPTION
**Pull request by jf-viguier**
numRows & Affected_Rows methods

### CODE REVIEW

File: src/content/1.7/development/database/db.md [hunk 1]
```diff
@@ -65,6 +65,19 @@ As the method deals with raw SQL requests, the `_DB_PREFIX_` must be used.
 This method is convenient when you need only one value to retrieve from the database.
 It prevent to loop in several arrays in order to get the first value of the first row.
 
+### Gets the number of rows in a result
+```php
+$db->numRows();
+```
+This method caches and returns the number of rows from the last result set. 
+Be carefull, if your request has a limit, numRows doesn't return the total rows, you have to make a SELECT COUNT(*).
```
Comment (Progi1984) [inline src/content/1.7/development/database/db.md:73]:
```suggestion
Be careful, if your request has a limit, the method `numRows()` doesn't return the total rows. If you need, you have to make a `SELECT COUNT(*)`.
```

File: src/content/1.7/development/database/db.md [hunk 2]
```diff
@@ -65,6 +65,19 @@ As the method deals with raw SQL requests, the `_DB_PREFIX_` must be used.
...
+
+### Gets the number of affected rows
+```php
+$db->Affected_Rows();
+```
+returns the number of rows impacted by the latest INSERT, UPDATE, REPLACE or DELETE query.
```
Comment (Progi1984) [inline src/content/1.7/development/database/db.md:79]:
```suggestion
Returns the number of rows impacted by the latest `INSERT`, `UPDATE`, `REPLACE` or `DELETE` query.
```

**Review by Progi1984 — APPROVED** [review]

File: src/content/1.7/development/database/db.md [hunk 3]
```diff
@@ -65,6 +65,19 @@ As the method deals with raw SQL requests, the `_DB_PREFIX_` must be used.
 This method is convenient when you need only one value to retrieve from the database.
 It prevent to loop in several arrays in order to get the first value of the first row.
 
+### Gets the number of rows in a result
+```php
+$db->numRows();
+```
+This method caches and returns the number of rows from the last result set. 
+Be careful, if your request has a limit, the method `numRows()` doesn't return the total rows. If you need, you have to make a `SELECT COUNT(*)`.
```
Comment (matks) [inline src/content/1.7/development/database/db.md:73]:
```suggestion

Be careful, if your request has a limit, the method `numRows()` is limited too. If you need to retrieve the number of rows without limits, you need to use `SELECT COUNT(*)`.
```

**Review by matks — COMMENTED** [review]
Suggestion but not blocking 😉

**Review by matthieu-rolland — COMMENTED** [review]
hi @jf-viguier , thank you for your contribution !

You only have one small non mandatory feedback and your PR is good to go 👍

**Comment by jf-viguier** [comment]
@matthieu-rolland done

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @jf-viguier

**Pull request merged**